docs/tp_templates/applications/sdm_app.md

```diff
@@ -0,0 +1,285 @@
+</figure>
+
+The application also includes a job monitoring page to track the status of the submitted jobs.
+
+<figure>
+  <img src="../img/sdm_app_template_with_job_dark.jpeg" class="visible-dark" />
+  <img src="../img/sdm_app_template_with_job_light.jpeg" class="visible-light"/>
+  <figcaption>Manage job with the Taipy scenario management application created by the scenario management template out-of-the-box.</figcaption>
+
+This template offers several key benefits:
+
+- **Accelerated Development**: By leveraging scenario and data management visual elements,
+    developers can quickly bootstrap a standard application, saving significant
+    development time to focus on delivering business value more efficiently.
+- **Comprehensive Features**: The template provides a best-practice folder scaffolding for a
+    scenario management application, which supports a wide range of functionalities, including
+    data and scenario visualization, job orchestration monitoring, providing a comprehensive
+    solution for various use cases.
+- **Customization**: The generated application is highly customizable with pre-built
+    placeholders for various components, allowing developers to tailor the application
+    to meet specific requirements and use cases.
+- **Deployment-Ready**: The template provides Docker deployment helpers for both development purpose
+    and production environment.
+
+# How to create an application
+
+To create the application from the scenario management template, change to the folder in which
+you want to create the application and run the command `taipy create --application sdm`. Then answer
+a few questions to customize your application.
+
+```console
+$ taipy create --application sdm
+[1/6] Application root folder [taipy_application]:
+[2/6] Application main Python file [main.py]:
+[3/6] Application title [Taipy Application]:
+[4/6] With TOML Config? (No):
+[5/6] With a new Git repository? (No):
+[6/6] Select With Docker deployment
+    1 - No
+    2 - For development
+    3 - For production
+    Choose from [1/2/3] (1):
+New Taipy application has been created at ./taipy_application
+
+To start the application, change directory to the newly created folder:
+    cd ./taipy_application
+
+You can then run the application as follows:
+    taipy run main.py
+```
+
+??? info "Default answers"
+
+    In the CLI, the default answer for each question is displayed in the square brackets.
+    You can provide an answer or press Enter to use the default value.
+
+Each question in the CLI corresponds to a specific aspect of the application. The following
+sections describe each question in detail.
+
+!!! note "Available in Taipy Enterprise edition"
+
+    Questions 6 is only relevant to the [Taipy Enterprise Edition](https://taipy.io/enterprise)
+
+    [Contact us](https://taipy.io/book-a-call){: .tp-btn .tp-btn--accent target='blank' }
+
+## 1. Application root folder
```

Comment:
I would turn the title into an enumeration to make the TOC lighter.

Reply:
I created a new issue for this: https://github.com/Avaiga/taipy-enterprise/issues/642